generation/maisi/scripts/diff_model_infer.py

```diff
@@ -59,13 +60,13 @@ def load_models(args: argparse.Namespace, device: torch.device, logger: logging.
     """
     autoencoder = define_instance(args, "autoencoder_def").to(device)
     try:
-        checkpoint_autoencoder = torch.load(args.trained_autoencoder_path)
+        checkpoint_autoencoder = torch.load(args.trained_autoencoder_path, weights_only=True)
         autoencoder.load_state_dict(checkpoint_autoencoder)
     except Exception:
-        logger.error("The trained_autoencoder_path does not exist!")
+        logger.info("The trained_autoencoder_path does not exist!")
```

Comment:
The same as [here](https://github.com/Project-MONAI/tutorials/pull/1839/files#r1772498249), cc @dongyang0122